charmed_openstack_info/data/lp-builder-config/ovn.yaml

```diff
@@ -61,6 +61,14 @@ defaults:
       bases:
         - "22.04"
         - "23.04"
+    stable/23.09:
+      build-channels:
+        charmcraft: "2.x/stable"
```

Comment:
this should be 2.x/candidate otherwise the mantic charms won't be built, charmcraft 2.5.0 hasn't made it into 2.x/stable yet